### PR TITLE
Bump taxpasta version

### PR DESCRIPTION
**Pull request by jfy133**
As on the tin

## PR checklist

Closes #XXX 

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Review by pontus — APPROVED** [review]

File: modules/nf-core/taxpasta/merge/main.nf [hunk 1]
```diff
@@ -4,8 +4,8 @@ process TAXPASTA_MERGE {
 
     conda "bioconda::taxpasta=0.2.3"
```
Comment (Midnighter) [inline modules/nf-core/taxpasta/merge/main.nf:5]:
```suggestion
    conda "bioconda::taxpasta=0.4.0"
```

File: modules/nf-core/taxpasta/standardise/main.nf [hunk 2]
```diff
@@ -4,8 +4,8 @@ process TAXPASTA_STANDARDISE {
 
     conda "bioconda::taxpasta=0.2.3"
```
Comment (Midnighter) [inline modules/nf-core/taxpasta/standardise/main.nf:5]:
```suggestion
    conda "bioconda::taxpasta=0.4.0"
```

File: modules/nf-core/taxpasta/merge/main.nf [hunk 3]
```diff
@@ -4,8 +4,8 @@ process TAXPASTA_MERGE {
...
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/taxpasta:0.2.3--pyhdfd78af_0':
-        'biocontainers/taxpasta:0.2.3--pyhdfd78af_0' }"
+        'https://depot.galaxyproject.org/singularity/taxpasta:0.4.0--pyhdfd78af_0 ':
```
Comment (Midnighter) [inline modules/nf-core/taxpasta/merge/main.nf:7]:
```suggestion
        'https://depot.galaxyproject.org/singularity/taxpasta:0.4.0--pyhdfd78af_0':
```

**Review by Midnighter — CHANGES_REQUESTED** [review]
You forgot the conda versions 😉 

Is the `pyhdfd78af` part already guaranteed? Or do we need to wait for the container to get confirmation on that?

**Comment by jfy133** [comment]
> You forgot the conda versions 😉 
> 
> Is the `pyhdfd78af` part already guaranteed? Or do we need to wait for the container to get confirmation on that?

The container was already built at 18:43 yesterday! I took the string directly from the galaxy depot website :)

**Review by Midnighter — APPROVED** [review]

**Pull request merged**